segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/bookkeeper/BookKeeperServiceRunner.java

```diff
@@ -171,6 +173,15 @@ public void startAll() throws Exception {
 
     private void initializeZookeeper() throws Exception {
         log.info("Formatting ZooKeeper ...");
+
+        if (this.secureZK) {
+            System.setProperty("zookeeper.client.secure", "true");
+            System.setProperty("zookeeper.clientCnxnSocket", "org.apache.zookeeper.ClientCnxnSocketNetty");
+            System.setProperty("zookeeper.ssl.trustStore.location", this.tlsTrustStore);
+            //TODO: Read from a config parameter
```

Comment:
No `TODO`s without an associated issue, see our contributing guidelines.

https://github.com/pravega/pravega/wiki/Contributing#todos

Reply:
This is already fixed. Removing `TODO`.